block_my_feedback.php

```diff
@@ -142,7 +142,7 @@ public function fetch_marking(stdClass $user): ?array {
                     $assess->cmid = $cmid;
                     $assess->modname = $mod->modname;
                     // Get due date and require marking.
-                    $assess = self::get_mod_data($mod, $assess);
+                    $assess = $this->get_mod_data($mod, $assess);
```

Comment:
Why don't we make `get_mod_data()` a static function, then leave this call as `self::get_mod_data($mod, $assess)`?